lib/src/git.rs

```diff
@@ -57,9 +57,16 @@ pub fn import_refs(
     git_repo: &git2::Repository,
 ) -> Result<(), GitImportError> {
     let store = mut_repo.store().clone();
-    let git_refs = git_repo.references()?;
     let mut existing_git_refs = mut_repo.view().git_refs().clone();
+    let mut old_git_heads = HashSet::new();
```

Comment:
Would it make sense to make a non-mut view of these?

```
let old_git_heads = existing_git_refs
    .values()
    .iter().map(|old_target| old_target.adds())
    .flatten()
    .collect();
```

Reply:
Thanks, that's much better (could even drop the `.iter()` and use `flat_map()`)